src/hex/coordinate.rs

```diff
@@ -29,6 +35,46 @@ pub enum HexDirection {
     FrontLeft,
 }
 
+impl From<i32> for HexDirection {
+    fn from(value: i32) -> Self {
+        match value % 6 {
```

Comment:
We can use `value.rem_euclid(6)` to eliminate the need to match on multiple values.

Reply:
Added this as a more elegant looking solution

Reply:
looks good